www/src/result.js

```diff
@@ -53,11 +53,15 @@ input.annotations.forEach((anno, id) => {
     anno.isOk = anno.isComplete || anno.exception === anno.spec;
   }
 
+  function anchorPrefix(id) {
+    return /^[1-9]/.test(id) ? 'section' : 'appendix';
```

Comment:
This logic doesn't match the rust version. So we need to handle the case where the ID doesn't match `a-z`,`A-Z`.